Source/SwiftLintFramework/Rules/Style/PreferZeroOverExplicitInitRule.swift

```diff
@@ -0,0 +1,76 @@
+import Foundation
+import SourceKittenFramework
+
+public struct PreferZeroOverExplicitInitRule: OptInRule, ConfigurationProviderRule, SubstitutionCorrectableRule,
+                                              AutomaticTestableRule {
+    public var configuration = SeverityConfiguration(.warning)
+    private var pattern: String {
+        let zero = "\\s*:\\s*0(.0*)?\\s*"
+        let type = "(\(["CGPoint", "CGSize", "CGVector", "CGRect"].joined(separator: "|")))"
+        let firstArg = "(\(["x", "dx", "width"].joined(separator: "|")))"
+        let secondArg = "(\(["y", "dy", "height"].joined(separator: "|")))"
+        let thirdAndFourthArg = "(\\,\\s*width\(zero)\\,\\s*height\(zero))?"
+        return "\(type)\\(\\s*\(firstArg)\(zero)\\,\\s*\(secondArg)\(zero)\(thirdAndFourthArg)\\)"
+    }
+
+    public static let description = RuleDescription(
+        identifier: "prefer_zero_over_explicit_init",
+        name: "Prefer Zero Over Explicit Init",
+        description: "Prefer `.zero` over excplicit init with zero parameters (e.g. `CGPoint(x: 0, y: 0)`)",
+        kind: .style,
```

Comment:
can you update this to be `.idiomatic` and move the rule to the `Idiomatic` folder?